### PR TITLE
DOC-359: Update c8ql tutorial

### PR DESCRIPTION
**Pull request by diana-macrometa**
- Rewrote the tutorial
- Broke it into seven parts
- Moved images into a subfolder

### CODE REVIEW

**Comment by elof** [comment]
@diana-macrometa looks like this branch is out of date with master. I'll hold off on reviewing until changes have been merged

**Comment by diana-macrometa** [comment]
@elof, branch is merged

**Comment by diana-macrometa** [comment]
@elof, don't worry if the branch is not up to date with this PR. That's normal. Just click the link. Nobody else is working on this topic, there will not be issues.

File: docs/c8ql/got-tutorial/import-and-merge.md
Comment (jamesm-macrometa) [inline docs/c8ql/got-tutorial/import-and-merge.md]:
```suggestion
If you were to rename or translate one trait, then you would need to find all other character documents with the same trait and perform the changes there too. If you only refer to a trait in another collection, then you just need to update one record in a single document.
```

Extra space

File: docs/c8ql/got-tutorial/import-and-merge.md
Comment (jamesm-macrometa) [inline docs/c8ql/got-tutorial/import-and-merge.md]:
```suggestion
The idea here is that _traits_ is supposed to store documents keys of another collection, which you can use to resolve the letters to labels such as `strong`. By using another collection for the actual traits, you can easily query for all existing traits later on and store labels in multiple languages for instance in a central place. If you embedded traits directly, it becomes unmanageable.
```

Maybe use code box tag for individual traits?  Not sure if it's really helpful, up to you.

File: docs/c8ql/got-tutorial/graph-traversal.md
Comment (jamesm-macrometa) [inline docs/c8ql/got-tutorial/graph-traversal.md]:
```suggestion
To return the parents and grandparents of Joffrey, you can walk edges in `OUTBOUND` direction and adjust the traversal depth to go at least 1 step, and 2 at most:
```

Typo

File: docs/c8ql/got-tutorial/graph-traversal.md
Comment (jamesm-macrometa) [inline docs/c8ql/got-tutorial/graph-traversal.md]:
```suggestion
It might be a bit unexpected that Joffrey is returned twice. However, if you look at the graph visualization, you can see that multiple paths lead from Joffrey (bottom right) to Tywin:
```

Comma not necessary, but I'm fine with leaving it if we want to make it clearer for ESL readers.

File: docs/c8ql/got-tutorial/graph-traversal.md
Comment (jamesm-macrometa) [inline docs/c8ql/got-tutorial/graph-traversal.md]:
```suggestion
The same result will be returned for Robb, Arya, and Sansa as starting point. For Jon Snow, it will only be Ned.
```

**Review by jamesm-macrometa — APPROVED** [review]
Great work!  This is looking much cleaner.  Just a few minor suggestions.

**Pull request merged**
